airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/source.py

```diff
@@ -117,42 +126,25 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         # 1. Basic streams:
         streams = [
             Advertisers(**args),
-            AdvertiserIds(**args),
```

Comment:
why was this one removed?

Reply:
AdvertiserIds - supposed to be 'internal' stream which is needed for production accounts .